src/valid/type.rs

```diff
@@ -137,7 +137,11 @@ impl super::Validator {
             crate::ScalarKind::Float => {
                 width == 4 || (width == 8 && self.capabilities.contains(Capabilities::FLOAT64))
             }
-            crate::ScalarKind::Sint | crate::ScalarKind::Uint => width == 4,
+            crate::ScalarKind::Sint | crate::ScalarKind::Uint => {
+                width == 4
+                    || (width == 1/* TODO?: && self.capabilities.contains(Capabilities::INT8)? */)
```

Comment:
I think having these capabilities would make sense, but nothing would be able to set them to true until the proper support is added to wgpu-hal backends